src/collective/importexport/browser/import_view.py

```diff
@@ -9,7 +10,22 @@
 from zope.annotation import IAnnotations
 from zope.globalrequest import getRequest
 from zope.schema import getFieldsInOrder
-from zope.schema.interfaces import IContextSourceBinder, IBool, IText, IBytes, IInt, IFloat, IDecimal, IChoice, IDate, IDatetime, ITime
+from zope.schema.interfaces import (
```

Comment:
Do I need to resort them? What are the rules, I am not 100% clear. Not alphabetically?